packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -6198,7 +6197,7 @@ Specifies visibility settings for the application
 | Property          | Description                                        | DataType                            | Nullable | Default | MinLength | MaxLength | Validation |
 | ----------------- | -------------------------------------------------- | ----------------------------------- | -------- | ------- | --------- | --------- | ---------- |
 | appLinks          | Displays specific appLinks for the app             | [AppLinks object](#applinks-object) | FALSE    |         |           |           |            |
-| <ApiLifecycle access="ea" /> autoLaunch  |  Automatically sign into the app when user signs into Okta.            | Boolean | FALSE   | FALSE    |           |           |            |
+| <ApiLifecycle access="ea" /> autoLaunch  |  Automatically signs into the app when user signs into Okta.            | Boolean | FALSE   | FALSE    |           |           |            |
```

Comment:
We should always use "sign in to <blah>" not "sign into", because "sign in" is the verb
> Automatically signs **in to** the app when user signs into Okta.